### PR TITLE
A proposal to add Matz to the Villains.

### PR DESCRIPTION
**Pull request by ic**
With a spooky-jerky image and comment!

### CODE REVIEW

File: app/views/welcome/index.html.erb [hunk 1]
```diff
@@ -24,6 +24,15 @@
   </header>
 
   <div class="box alt container">
+    <section id="the-hidden-samurai" class="feature right">
```
Comment (davy) [inline app/views/welcome/index.html.erb]: this should have the 'left' class instead of 'right' 🐞

**Comment by ic** [comment]
@davy Thanks for pointing that out. Fixed.

**Comment by davy** [comment]
That picture is amazing, by the way 👌

**Comment by ic** [comment]
Cheers. The public domain is amazing :-)

**Pull request merged**